code/src/main/kotlin/com/expediagroup/sdk/core/http/Headers.kt

```diff
@@ -158,4 +158,17 @@ class Headers private constructor(private val headersMap: Map<String, List<Strin
             return Headers(LinkedHashMap(headersMap))
         }
     }
+
+    companion object {
+        @JvmStatic
+        fun builder(headers: Headers): Builder = Builder(headers)
+
+
+        @JvmStatic
+        fun builder(): Builder = Builder()
+
+        private fun sanitizeName(value: String): String {
+            return value.lowercase(Locale.US).trim()
```

Comment:
You've added .trim(), but I don't see a test that would fail if you removed it.

Reply:
Nice catch, added the test [here](https://github.com/ExpediaGroup/lodging-connectivity-java-sdk/pull/115/commits/00da60e0cf435ba28b4ae330668a5cdab885b71c)